src/Controller/Rest/AbstractRestController.php

```diff
@@ -50,7 +50,7 @@ public function needsSessionDoubleAuthenticationCheck(): bool
     /**
      * @inheritDoc
      */
-    protected function checkPermission($permission)
+    protected function checkPermission(string $permission): void
```

Comment:
```suggestion
    protected function checkPermission($permission): void
```
Otherwise it breaks on 10.6 version.